code/__HELPERS/global_lists.dm

```diff
@@ -97,7 +97,7 @@ GLOBAL_LIST_EMPTY(randomized_pill_icons)
 			"minimap" = icon2base64(xeno_minimap),
 			"sort_mod" = per_tier_counter[tier]++,
 			"tier" = GLOB.tier_as_number[tier],
-			"is_unique" = tier == XENO_TIER_FOUR, //TODO: Make this check a flag after caste flag refactoring is merged.
+			"is_unique" = initial(caste.maximum_active_caste) == 1,
```

Comment:
Does maximum_active_caste ever change?

Reply:
Might be admin fuckery, but since the other caste variable check uses initial() too, so I did uses it here as well, should I not?

Reply:
i dont see an advantage to using initial here

Reply:
oh, I'll remove it tommorow then